.github/workflows/format-check.sh

```diff
@@ -1,3 +1,11 @@
+# Copyright (C) 2020-2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier:ISC
+
 #! /bin/bash
```

Comment:
Please add a check that makes sure the hashbang line is the first line in the file.

Reply:
https://github.com/SymbiFlow/actions/pull/29